<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcquisitionWidgetVerticalSimpleLayout</class>
 <widget class="QWidget" name="AcquisitionWidgetVerticalSimpleLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>229</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AcquisitionWidgetVerticalSimple</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="acq_gbox">
     <property name="maximumSize">
      <size>
       <width>32767</width>
       <height>32767</height>
      </size>
     </property>
     <property name="title">
      <string>Acquisition</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QLabel" name="num_images_label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>135</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>50</weight>
                  <bold>false</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="text">
                 <string>Number of images:</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="osc_start_label">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Oscillation start:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="exp_time_label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Exposure time (s):</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="kappa_label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Kappa:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="energy_label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Energy (KeV):</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="resolution_label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Resolution (Ã…):</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="transmission_label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Transmission (%):</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <widget class="QComboBox" name="num_images_cbox">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>1 Image</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2 Images</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>4 Images</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="osc_start_ledit">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="exp_time_ledit">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="kappa_ledit">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="energy_ledit">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="resolution_ledit">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="transmission_ledit">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>27</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="osc_range_label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Oscillation range:</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="detector_roi_mode_label">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Detector mode:</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="kappa_phi_label">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="text">
                 <string>Phi:</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>27</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="osc_range_ledit">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="detector_roi_mode_combo">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="kappa_phi_ledit">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
